test/nodejs/search.js

```diff
@@ -1332,9 +1323,17 @@ describe('Gix Position Tests', function () {
             assert.equal(base.search({ $from: "TestStore", Value: "planica tralala" }).length, 0);
             assert.equal(base.search({ $from: "TestStore", Value: "tralala planica" }).length, 0);
             assert.equal(base.search({ $from: "TestStore", Value: "aa aa" }).length, 1);
-            assert.equal(base.search({ $from: "TestStore", Value: "aa aa" })[0].$fq, 6);
+            assert.equal(base.search({ $from: "TestStore", Value: "aa aa" })[0].$fq, 8);
             assert.equal(base.search({ $from: "TestStore", Value: "aa bb" }).length, 1);
-            assert.equal(base.search({ $from: "TestStore", Value: "aa cc" }).length, 0);
+            assert.equal(base.search({ $from: "TestStore", Value: "aa cc" }).length, 1);
```

Comment:
Example would be query for "aa bb" in "aa aa bb aa aa aa bb aa" where "aa" is split over two items and "bb" is in one. Reverse would be query for "aa bb" in  "aa bb bb bb bb aa bb".